### PR TITLE
Add NuGet README file

### LINKED ISSUE #5899: Create README for NuGet packages
Add README to Microsoft.ML package to display more information on NuGet.

Include information like:

- Link to documentation

https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/

### PR DESCRIPTION
**Pull request by luisquintanilla**
We are excited to review your PR.

So we can do the best job, please check:

- [x] There's a descriptive title that will make sense to other developers some time from now. 
- [x] There's associated issues. All PR's should have issue(s) associated - unless a trivial self-evident change such as fixing a typo. You can use the format `Fixes #nnnn` in your description to cause GitHub to automatically close the issue(s) when your PR is merged.
- [x] Your change description explains what the change does, why you chose your approach, and anything else that reviewers should know.
- [x] You have included any necessary tests in the same PR.

Fixes #5899

### CODE REVIEW

File: src/Microsoft.ML/Microsoft.ML.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <!-- <IncludeBuildOutput>false</IncludeBuildOutput> -->
     <NoWarn>$(NoWarn);NU5127;NU5128</NoWarn>
     <PackageDescription>ML.NET is a cross-platform open-source machine learning framework which makes machine learning accessible to .NET developers.</PackageDescription>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```
Comment (michaelgsharp) [inline src/Microsoft.ML/Microsoft.ML.csproj:12]: This isn't the correct path to the readme file. Thats why the builds are failing. 

I think prefixing with `../..` will fix the problem.

**Review by michaelgsharp — CHANGES_REQUESTED** [review]
Sorry, been OOF but I'm back now. Path needs fixing, but otherwise I think its good.

File: src/Microsoft.ML/Microsoft.ML.csproj (hunk 1 above)
Comment (luisquintanilla) [reply src/Microsoft.ML/Microsoft.ML.csproj]: Thanks. I just made the suggested update.

Comment (michaelgsharp) [reply src/Microsoft.ML/Microsoft.ML.csproj:12]: It looks like the latest change removed it again. Did it not fix it?

**Comment by chgill-MSFT** [comment]
@luisquintanilla What version of the .NET SDK are you using to pack? We had an [issue](https://github.com/NuGet/Home/issues/10791) that didn't allow README inclusion when using snupkg for symbols, but it was fixed in .NET 6 RC 2.

**Comment by luisquintanilla** [comment]
I believe ML.NET targets .NET Standard 2.0. @michaelgsharp would know better though.

**Comment by michaelgsharp** [comment]
The PR that updated us to use the fixed version of .NET 6 just went in yesterday. I have fixed the include to point to the right readme file and this now works as expected. Once the builds pass I can merge this in.

**Review by michaelgsharp — APPROVED** [review]
:shipit: LGTM.

**Pull request merged**